### DISCUSSION TITLE
GPT-4o Compatiblity?

### DISCUSSION THREAD

**Discussion by omartan** [zed-industries/zed#11779] [Feature Requests]
Hi, 

It seems OpenAI released a new cost-effective modal? Any idea when we will see it in Zed?

<img width="1013" alt="Screenshot 2024-05-14 at 9 38 02 AM" src="https://github.com/zed-industries/zed/assets/12379413/a185844d-26b9-4afd-ae05-98299fbb1ae6">

**Comment by mrnugget** [comment]
Just merged: https://github.com/zed-industries/zed/pull/11764
